pkgs/by-name/sr/srgn/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "srgn";
+  # the latest versioned release has broken unit tests, so we use the latest commit instead
```

Comment:
We fixed this is https://github.com/alexpovel/srgn/issues/66 , so this wouldn't be necessary anymore.